RInAppMessaging/Classes/Utilities/KeyHasher.swift

```diff
@@ -0,0 +1,79 @@
+import CommonCrypto
+import CryptoKit
+
+/// Tool for creating a salted hash from given string values using selected KeyHasher.Encryption method
+internal struct KeyHasher {
+
+    enum Encryption: String {
+        case md5
+        case sha256
+    }
+
+    var encryprtionMethod = Encryption.sha256
```

Comment:
encryprtionMethod -> encryptionMethod

Reply:
I like AndroidStudio more because it checks for typos...